tests/generate_test.js

```diff
@@ -134,6 +154,24 @@ ${formatStyles(actual)}
         }], '.foo{color:blue !important;}');
     });
 
+    it('does not mutate nested objects', () => {
```

Comment:
Note: this test exposes a bug that is currently in master that this commit fixes.